azkaban-common/src/main/java/azkaban/project/ProjectManager.java

```diff
@@ -117,6 +119,7 @@ public boolean hasFlowTrigger(final Project project, final Flow flow)
 
   private void loadAllProjects() {
     final List<Project> projects;
+    logger.info("Loading active projects.");
```

Comment:
Do you think it will be more clear if we print out the time for the loading process instead? Something like:
```
t1 = System.currentTimeMillis();
// loading active projects
t2 = System.currentTimeMillis();
logger.info("Time to load active projects is" + (t2-t1));
```

Reply:
Sounds like a good idea. Gives relevant information.

Reply:
We can look at the timestamps in the logs to get those times.
I would consider adding time calculations if we were going to take some action in code for example if the delay goes above some threshold.